var/spack/repos/builtin/packages/py-fastcov/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyFastcov(PythonPackage):
+    """
+    A parallelized gcov wrapper for generating intermediate coverage formats
+    fast
+    """
+
+    homepage = "https://github.com/RPGillespie6/fastcov"
+    pypi     = "fastcov/fastcov-1.13.tar.gz"
+
+    maintainers = ['haampie']
+
+    version('1.13', sha256='ec8a5271f90a2f8b894cb999e262c33e225ed6072d9a6ca38f636f88cc0543e8')
+
+    # Depends on gcov too, but that's installed with gcc
```

Comment:
Does this mean it only supports GCC?

Reply:
Hm, let me rephrase that to `with the compiler`, cause I think llvm also has a gcov, but a different codecov format.

Reply:
```suggestion
    # Depends on gcov too, but that's installed with the compiler
```